ivy/functional/frontends/tensorflow/math.py

```diff
@@ -137,3 +137,22 @@ def reduce_std(input_tensor, axis=None, keepdims=False, name="reduce_std"):
 
 def asinh(x, name="asinh"):
     return ivy.asinh(x)
+
+
+def reduce_variance(input_tensor, axis=None, keepdims=False, name="reduce_variance"):
+    return ivy.var(input_tensor, axis=axis, keepdims=keepdims)
+
+
+reduce_variance.unsupported_dtypes = {
```

Comment:
You shouldn't be setting this many unsupported dtypes, almost all of the floats are filtered out in most of the backends, this combined with it only testing for floats, will cause test to not run this example

Reply:
Thank you so much for this review @Unevilicorn. I will make the requested changes right away.

Reply:
@Unevilicorn I have reduced the unsupported types and opened it for another review.